src/app.py

```diff
@@ -1029,11 +1029,9 @@ def create_entity(entity_type):
         # Also check existence of the previous revision dataset if specified
         if 'previous_revision_uuid' in json_data_dict:
             if isinstance(json_data_dict['previous_revision_uuid'], list):
+                if len(json_data_dict['previous_revision_uuid']) > 1:
```

Comment:
This `previous_revision_uuid` should always be a string, there's really no point to treat it as a list. Looking back I found this commit: https://github.com/hubmapconsortium/entity-api/commit/f37aa8dbc5c78d2c89bdc436d23aedfb39aa357a#diff-cd8a3bff2418f86ca995d983b9e0d2ed942f3a2a10b8eb393b15f18f56ea3fb4R421-R423 

We should remove the `- list` from yaml.